spaces/S000082/properties/P000062.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000082
+property: P000062
+value: false
+refs:
+- mathse: 3155216
+  name: "Example: limit point compact + ¬countably compact+ ¬Lindelöf"
+---
+
+The clopen cover using each of the uncountable copies of {S42} has no countable subcover.
```

Comment:
As written, this is the justification that the space is not Lindelof.  For weakly Lindelof, need to modify a little.